### DISCUSSION TITLE
Cleanlab support for text/markdown note/document?

### DISCUSSION THREAD

**Discussion by vasnt** [cleanlab/cleanlab#732] [Q&A] [answered]
Cleanlab can find duplicate data within a text/markdown note/document?

**Comment by jwmueller** [comment, marked as answer]
Yes. If you break the text into reasonably-sized chunks, use a model to obtain vector embeddings for each chunk, and then pass them as `features` into Datalab, it will automatically find nearly (and exactly) duplicated chunks for you. Learn more about Datalab here:

https://docs.cleanlab.ai/stable/tutorials/datalab/text.html
https://docs.cleanlab.ai/stable/tutorials/datalab/datalab_quickstart.html

(it will also detect other issues in your data like outliers)
